examples/camera-app/linux/include/camera-device.h

```diff
@@ -41,6 +41,9 @@ static constexpr uint32_t kMaxContentBufferSizeBytes = 4096;
 static constexpr uint32_t kMaxNetworkBandwidthMbps   = 128;
 static constexpr uint8_t kMaxConcurrentEncoders      = 1;
 static constexpr uint32_t kMaxEncodedPixelRate       = 27648000; // 720p at 30fps
+static constexpr uint8_t kSpeakerMinLevel            = 1;
+static constexpr uint8_t kSpeakerMaxLevel            = 254; // Spec constraint
+static constexpr uint8_t kSpeakerMaxChannelCount     = 8;   // Same as Microphone
 static constexpr uint8_t kMicrophoneMinLevel         = 1;
 static constexpr uint8_t kMicrophoneMaxLevel         = 254;  // Spec constraint
```

Comment:
The Max for the ImageRotation needs to be defined here.

Reply:
Added constants for max and min